README.md

```diff
@@ -60,20 +60,49 @@ Follow the instructions below, or watch the [installation video](https://vimeo.c
 
 #### Obtaining a licence key
 
 New users can register for a free AddressFinder account at one of these links:
-* [Free Account for Australia](https://portal.addressfinder.io/signup/au/free)
-* [Free Account for New Zealand](https://portal.addressfinder.io/signup/nz/free)
+* [Free Account for Australia](https://portal.addressfinder.io/signup/au/free?utm_source=bigcommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=Free%20Account%20for%20Australia&utm_content=Obtaining%20a%20licence%20key)
+* [Free Account for New Zealand](https://portal.addressfinder.io/signup/nz/free?utm_source=bigcommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=Free%20Account%20for%20New%20Zealand&utm_content=Obtaining%20a%20licence%20key)
 
 Existing users can obtain their API key from the
-[AddressFinder Portal](https://portal.addressfinder.io/).
+[AddressFinder Portal](https://portal.addressfinder.io/?utm_source=bigcommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=AddressFinder%20Portal&utm_content=Obtaining%20a%20licence%20key).
+
+### Advanced Usage
+
+If you want to adjust the default behaviour of the widget you may add additional options.These options should be added inside an object with the key 'widgetOptions' and should be in the form of key value pairs. This section may be excluded for default behaviour. For a full list of possible options visit our [Widget Documentation](https://addressfinder.nz/docs/widget_docs/?utm_source=bigcommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=Widget%20Documentation&utm_content=Advanced%20Usage)
+
+```
+  w.AddressFinderPlugin = {
+    key: "ADDRESSFINDER_NZ_DEMO_KEY",
+    widgetOptions: {
+      byline: false
+    }
+  };
+```
+### Country Specific Options
+If you need to add different options for each country you can enter them in objects with the keys 'nzWidgetOptions' and 'auWidgetOptions'.
+
+```
+  w.AddressFinderPlugin = {
+    key: "ADDRESSFINDER_NZ_DEMO_KEY",
+    nzWidgetOptions: {
+      byline: false
+    },
+    auWidgetOptions: {
+      byline: true
+    }
+  };
+```
+
+For the full list of [NZ specific options](https://addressfinder.nz/docs/widget_docs/?utm_source=bigcommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=Widget%20Documentation&utm_content=Country%20Specific%20Options)
+For the full list of [AU specific options](https://addressfinder.com.au/docs/widget_docs/?utm_source=bigcommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=Widget%20Documentation&utm_content=Country%20Specific%20Options)
 
 ## Software License
 
-The AddressFinder plugin for Salesforce is released under the permissive free software [MIT License](https://github.com/AbleTech/addressfinder-bigcommerce/blob/master/LICENCE.md).
\ No newline at end of file
+The AddressFinder plugin for Salesforce is released under the permissive free software [MIT License](https://github.com/AbleTech/addressfinder-bigcommerce/blob/master/LICENCE.md).
```

Comment:
Can you rename "SalesForce" to "BigCommerce" please? Also do a search throughout the code base for any other mentions of SalesForce.